extensions/geode-modules/src/main/java/org/apache/geode/modules/gatewaydelta/GatewayDelta.java

```diff
@@ -16,9 +16,9 @@
 
 public interface GatewayDelta {
 
-  public static final String GATEWAY_DELTA_REGION_NAME = "__gatewayDelta";
+  String GATEWAY_DELTA_REGION_NAME = "__gatewayDelta";
```

Comment:
do we want to remove static final?

Reply:
I believe any field in an `interface` is implicitly `public static final`.

Even without the `final`, I get a compile error in a small test class that attempts to modify the value.

My `tmp.java`:
```
package org.apache.geode.modules.gatewaydelta;
public class tmp {
  static {
    GatewayDelta.GATEWAY_DELTA_REGION_NAME = "null";
  }
}
```
produces
```
:20: error: cannot assign a value to final variable GATEWAY_DELTA_REGION_NAME
    GatewayDelta.GATEWAY_DELTA_REGION_NAME = "null";
```

But I'll admit, I just had IntelliJ's inspections do all the legwork, so it's a good question to ask.

Reply:
good to know